### ISSUE TITLE
Typo in SoftMax docs prevent formula rendering

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#3186]
(removed previous report addressed by https://github.com/pytorch/pytorch/pull/3245)

There's a missing letter 'f' in the [docs](http://pytorch.org/docs/master/nn.html?highlight=softmax#torch.nn.functional.softmax), my browser doesn't render the equation:
```
:math:`softmax(x) =

rac{exp(-x_i)}{sum_j exp(-x_j)}`
```

**Comment by soumith** [comment]
fixed via #3327
